### ISSUE TITLE
Getting invalid argument --teamcity error

### ISSUE THREAD

**Issue by adeel41** [nunit/dotnet-test-nunit#72]
framework: netcoreapp1.0 
nunit: 3.4.1
dotnet-test-nunit: 3.4.0-beta-2

when I run my tests on command line using command `dotnet test --teamcity`, I keep getting this error message "Invalid argument: --teamcity". Any suggestions?

**Comment by jasonwilliams200OK** [comment]
`--teamcity` is not supported by the runner. See #4 for related discussion.

**Comment by rprouse** [comment]
We may decide to support TeamCity in a later release. Until then, you should be able to run your tests within TeamCity without the command line, you just won't get the rich integration that TeamCity provides.

**Comment by adeel41** [comment]
Thanks for the info.  I added a Command Line build step to run ```nunit3-console.exe``` with ```--teamcity``` argument to run the tests; which did the trick.

**Comment by brvaland** [comment]
Hi Adeel

Can I run nunit3-console.exe with --teamcity argument on teamcity docker image both server and agent running on docker ?

Please advise ?

**Comment by CharliePoole** [comment]
@brvaland Yes you can, if you have set it up correctly, including the extension that supports the argument. 

Note: this issue is about dotnet-test-nunit, which is an entirely different piece of software from the NUnit console runner.

**Comment by brvaland** [comment]
I would appreciate if someone could share on how to setup nunit3-console on Linux.

**Comment by CharliePoole** [comment]
@brvaland You can use the zip file from the nunit-console release to set up your image. Simply locate the console program someplace convenient so that you can execute it. https://github.com/nunit/nunit-console/releases

As noted, this is off-topic in the dotnet-test-nunit project, which is about the dotnet test runner that is used to execute .NET core tests only. As a courtesy to all of us, please post issues about the nunit3-console runner on the nunit-console project. If you prefer a more general place to ask questions, use the nunit-discuss group on google, which exists for that purpose.

**Comment by NikolayPianikov** [comment]
@brvaland you can run NUnit tests on mono. Try downloading NUnit into the TeamCity agent and add executably privilegies to `nunit3-console.exe`, `nunit-agent.exe`, 'nunit-agent-x86.exe'

**Comment by dmitriyse** [comment]
Try to use dotnet-test-nunit-teamcity package. It allows --teamcity switch. You need to modify project.json like this:
```json
{
  "version": "0.1.0-*",
  "dependencies": {
    "Microsoft.NETCore.App": {
      "type": "platform",
      "version": "1.1.0"
    },
    "Microsoft.Net.Test.Sdk": {
      "version": "15.0.0-preview-20161216-01"
    },
    "dotnet-test-nunit-teamcity": "3.4.0-beta-3",
    "NUnit": "3.5.0"
  },
  "testRunner": "nunit-teamcity",
  "frameworks": {
    "netcoreapp1.1": {
      "imports": "dnxcore50"
    }
  }
}

```

**Comment by rprouse** [comment]
Fixed by #95
